src/services/services.ts

```diff
@@ -3798,6 +3798,10 @@ module ts {
                 // Try getting just type at this position and show
                 switch (node.kind) {
                     case SyntaxKind.Identifier:
+                        if(isLabelName(node)){

```

Comment:
Put a space after the `if` and before the `{` @GuillaumeSalles.

@CyrusNajmabadi why? There is no type for a label name.